application/src/main/java/org/togetherjava/tjbot/features/moderation/TransferQuestionCommand.java

```diff
@@ -33,6 +33,16 @@
 import java.util.function.Supplier;
 import java.util.regex.Pattern;
 
+/**
+ * This command can be used to transfer questions asked in any channel to helper forum. Transfer
+ * action generates a prefilled modal menu with title and contents of message which can be changed
+ * by the user of command. On submission, it creates a new post in helper forum(also mentions the
+ * author) followed by sending a dm to user and deleting the original message from chat. If dm
+ * fails(user blocked tj-bot), user is notified in same channel.
+ * <p>
+ * The command fails if user lacks permissions, or if used on message from a bot.
+ */
+
```

Comment:
To specific to the implementation. Descriptions that are so specific have to problems:
* its easier to read the exact details by "just reading the code" (provided its well written)
* it easily rotts and then doesnt reflect the truth anymore (just a tiny change on the code and forgetting to adjust the doc and boom, bad)

Its better to just describe the feature on a broader level, just giving a general overview of what its about. So that someone who opens the file and has never heard of "transfer command", generally understands what its all about.

```suggestion
/**
 * This command can be used to transfer questions asked in any channel to the helper forum.
 * The user is given the chance to edit details of the question and upon submitting,
 * the original message will be deleted and recreated in the helper forum.
 * The original author is notified and redirected to the new post.
 */

```